auction_seller_invoice/views/auction_item_invoice_views.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- pylint:disable=deprecated-data-xml-node -->
+<odoo>
+    <record id="account_move_seller_form_view" model="ir.ui.view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@id='other_tab']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath
+                expr="//group[@id='header_right_group']/field[@name='date']"
+                position="attributes"
+            >
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//label[@for='invoice_payment_term_id']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath
+                expr="//field[@name='invoice_payment_term_id']"
+                position="attributes"
+            >
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//label[@for='ref']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='payment_reference']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='partner_bank_id']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='invoice_date_due']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='amount_residual']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='activity_ids']" position="replace" />
+            <xpath expr="//field[@name='message_ids']" position="replace" />
+            <xpath
+                expr="//form/div[@class='o_attachment_preview']"
+                position="replace"
+            />
+            <xpath expr="//form/div[@class='oe_chatter']" position="replace" />
```

Comment:
For the sake of consistency, any reason to use `replace` here while other fileds are made invisible?

By the way, I think it's OK to use `replace` when the mode is `primary`, if I'm not mistaken.  We may actually use `replace` for all fields to make the changes more concise.

Reply:
I used `replace` for some fields because I need to give access right to auction seller user for the models of these fields (eg.mail.activity) if I use `invisible`. So, I don't want to give access right for unnecessary model and used `replace`.

Reply:
Yes. I will